### ISSUE TITLE
Unable to connect to any peers since 1.4.9 on Windows 7 x64

### ISSUE THREAD

**Issue by D-Nice** [ethcore/parity#4512]
I've been on Parity 1.4.9 for a while, and all of a sudden recently, it stopped being able to sync to any peers. I attempted an updated to Parity 1.5 and the same issue occurred. When I checked my clock with time.is on yesterday's tests, it was off by a couple of ms. I resync'd the clock before running parity with the log file, and time.is indicated it was 1.2 seconds off.

Here is the log file requested by @arkpar 

[parity.txt](https://github.com/ethcore/parity/files/767191/parity.txt)

**Comment by rphmeier** [comment]
> A request to send or receive data was disallowed because the socket is not connected and (when sending on a datagram socket using a sendto call) no address was supplied. (os error 10057)

On every single read call seems wrong. We just get these sockets from the mio UDP listener AFAIK.

**Comment by arkpar** [comment]
Looks like something is blocking all connections. Are you using any kind of firewall or anti-virus software that might have caused this? Try using a different port. E.g. `--port 40404`

**Comment by D-Nice** [comment]
Yes, I'm using the exact firewall/AV I've always used. I did not see it blocking parity, and I even had turned it off and the same result. I am suspicious of my ISP blocking this, which was my initial thought. They like to throttle any P2P traffic, however, I've even tried using parity with a VPN which circumvents this throttling, and still the same issue. Switching to port 40404 results in the same issue.

**Comment by D-Nice** [comment]
This was my fault. A few weeks back my network seemed to be under some form of DDoS attack, and I had blocked a number of ports in the router firewall. Most of the source and destination ports were 30303 so I had blocked those. Both outgoing and incoming were disabled. 

I did think changing the port would at least allow me to connect to a few peers in this case, regardless of the firewall blocking 30303. 

Anyways, issue is resolved and was indeed on my end, thanks for the help!
